packages/platforms/delta/x86-64/x86-64-delta-ag8032/modules/builds/delta-at24.c

```diff
@@ -0,0 +1,870 @@
+/*
```

Comment:
It appears that this is just a copy of the existing at24 driver already in the kernel. 
Is there a reason we need this one?

Reply:
the previous kernel config for the at24 driver is the  CONFIG_EEPROM_AT24=y, but now you change it into CONFIG_EEPROM_AT24=m in the 4.9.75 version. so it means that it was inserted manually, but the problem is that the building kernel module image at24.ko can not push into rootfs in the image, so i have to copy it into platform module to get the driver image to insert it, which it is what i wondered.

Reply:
@shaohuaxiong The at24 driver must remain a module from now on. I will investigate why it is not available in the RFS and then you can remove this copy.

Reply:
@jnealtowns The team has removed the at24 module, can you please accept this commit, or let us know if we are missing anything.

Reply:
the module is necessary , what' s mean for your removing at24 module. can you give more information?